src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is the most commonly used approach. This may not be the right approach if a strong application service continuity during the migration is required. However, it may be a viable method if downtime is permitted. It works particularly well when the application accurately maintains [primary key](primary-key.html) and [foreign key](foreign-key.html) constraints.
+
+## Pros and Cons
+
+On the spectrum of different data migration strategies, 'lift and shift' has the following pros and cons. The terms "low" and "high" are not absolute, but relative to other approaches.
+
+Pros:
+
```

Comment:
We should add something like if your application already has regularly scheduled maintenance windows no customers will encounter application downtime

Reply:
Nice, good point.  Added that!